docker-magento2/docs/INSTALLATION.md

```diff
@@ -0,0 +1,182 @@
+# Manual de instalación para Plugin Magento2
```

Comment:
If this is new/updated please remember to update it the transbank-developers-doc repository too

Reply:
This doc is only for developers, is a help to start up environment of magento2